### PR TITLE
Implemented Devise Gem

### PR DESCRIPTION
**Pull request by BlazeDex**
Import Devise Gem into Gemfile for:

- [x] User authentication.
- [x] Log in first view for e-mail and password.

### CODE REVIEW

**Comment by BlazeDex** [comment]
image

**Review by juperameza — APPROVED** [review]

**Review by MauricioNC — APPROVED** [review]

File: app/views/devise/confirmations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: confirmation_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
```
Comment (JuanequeX) [inline app/views/devise/confirmations/new.html.erb]: Remember to avoid using `<br />` when you apply your styles.

**Review by JuanequeX — APPROVED** [review]

**Review by Antonio3veces3 — APPROVED** [review]
Well done!

File: app/views/devise/confirmations/new.html.erb (hunk 1 above)
Comment (BlazeDex) [reply app/views/devise/confirmations/new.html.erb]: Fixed!

Comment (JuanequeX) [reply app/views/devise/confirmations/new.html.erb]: It was fixed only in this view, but in the others?

**Pull request merged**